etc/examples/paths.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# these can be set by folks running the examples/Makefile.testing to use a different binaries than in the path
+ECHO=${ECHO:-echo}
+JQ=${JQ:-jq}
+KUBECTL=${KUBECTL:-kubectl}
+PACHCTL=${PACHCTL:-pachctl}
+RM=${RM:-rm}
+SLEEP=${SLEEP:-sleep}
+TAR=${TAR:-tar}
```

Comment:
What platform-specific stuff is being used for tar?

Reply:
Same armoring as above.  This makes it easier to switch versions in a tool that could have cross-platform subtleties.